tests/acceptance/features/webUIFavorites/unfavoriteFile.feature

```diff
@@ -46,58 +50,51 @@ Feature: Unmark file/folder as favorite
-    When the user browses to the files page
-    Then file "data.zip" should not be marked as favorite on the webUI
-    But file "data.tar.gz" should be marked as favorite on the webUI
-    And file "lorem.txt" should be marked as favorite on the webUI
 
   Scenario: unmark a folder as favorite from favorite page
     Given user "user1" has favorited element "simple-folder"
     And user "user1" has favorited element "simple-empty-folder"
-    And user "user1" has favoritedser element "0"
+    And user "user1" has favorited element "0"
     And the user has browsed to the favorites page
     When the user unmarks the favorited folder "simple-folder" using the webUI
     Then folder "simple-folder" should be listed on the webUI
-    But folder "simple-folder" should not be marked as favorite on the webUI
-    When the user reloads the current page of the webUI
-    Then folder "simple-folder" should not be listed on the webUI
+    And folder "simple-folder" should not be marked as favorite
+    And folder "simple-folder" should not be marked as favorite on the webUI
     But folder "0" should be listed on the webUI
     And folder "simple-empty-folder" should be listed on the webUI
-    When the user browses to the files page
-    Then folder "simple-folder" should not be marked as favorite on the webUI
-    But folder "simple-empty-folder" should be marked as favorite on the webUI
-    And folder "0" should be marked as favorite on the webUI
 
   Scenario: unmark files/folders as favorites using the sidebar
     Given user "user1" has favorited element "data.zip"
     And user "user1" has favorited element "simple-folder"
+    And the user has browsed to the favorites page
     When the user unmarks the favorited folder "simple-folder" using the webUI sidebar
     And the user unmarks the favorited file "data.zip" using the webUI sidebar
     Then folder "simple-folder" should not be marked as favorite on the webUI
+    And folder "simple-folder" should not be marked as favorite
     And file "data.zip" should not be marked as favorite on the webUI
+    And file "data.zip" should not be marked as favorite
 
   @issue-1720
   Scenario: Try to unfavorite file and folder that used to exist but does not anymore
-    Given the user has browsed to the files page
-    And user "user1" has favorited element "simple-folder"
+    Given user "user1" has favorited element "simple-folder"
     And user "user1" has favorited element "lorem.txt"
+    And the user has browsed to the files page
     And the following files have been deleted by user "user1"
       | name          |
       | lorem.txt     |
       | simple-folder |
     When the user unmarks the favorited file "lorem.txt" using the webUI
     Then no message should be displayed on the webUI
     #    Then the error message with header 'Error while marking "lorem.txt" as favorite' should be displayed on the webUI
-    And file "lorem.txt" should not be marked as favorite on the webUI
```

Comment:
Since the unfavorite action doesn't go through we need to expect that the star in the sidebar is still active. At least until we fix the original issue.